packages/@aws-cdk/aws-stepfunctions-tasks/test/eks/integ.call.ts

```diff
@@ -0,0 +1,28 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { EksCall, MethodType } from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-eks-call-integ');
```

Comment:
this test is not self contained. Please include verification steps, but more importantly create the EKS cluster as a part of the integration test. 

The test should create a state machine that can be executed successfully. Right now it has a dependency that an EKS job already exists called `clusterName`, but this is not self-contained.

Verification steps should look something like [this](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-stepfunctions-tasks/test/lambda/integ.invoke.ts#L6-L17)

Reply:
Modified integ test in the latest commit